src/containers/AdminExternalSystems.tsx

```diff
@@ -222,54 +221,57 @@ class AdminExternalSystems extends Component<Props, State> {
           variant="contained"
           endIcon={<RefreshIcon />}
           onClick={this.handleRefreshSystems}
+          style={{ marginTop: 15, marginBottom: 15 }}
         >
-          Refresh
+          Refresh Lists
```

Comment:
Organizing suggestion to make it as clear as we can it's the one list of integrations that are getting refreshed, not the integrations themselves

```suggestion
          Refresh List
```